sigs.yml

```
@@ -466,6 +466,20 @@
     gcLiaison:
     - name: 'Juraci Paixão Kröhling'
       github: jpkrohling
+  - name: 'GoLang: Compile Time Instrumentation'
+    meeting: Every other Thursday at 08:00 UTC
+    notes:
+      type: gDoc
+      value: 1XkVahJfhf482d3WVHsvUUDaGzHc8TO3sqQlSS80mpGY
+    chat:
+      - type: slack
+        name: '#otel-go-compt-instr'
```

I don't think it's too long to spell out for clarity (or maybe `#otel-go-compile-instr` if shorter is desired)
```suggestion
        name: '#otel-go-compile-time-instr'
```